### ISSUE TITLE
[Input] Required file input shows as an error field on page load

### ISSUE THREAD

**Issue by gfkateo** [fomantic/Fomantic-UI#2749]
# Bug Report
When adding a `required` attribute in the file input element, the input field automatically shows as error on page load. 

## Steps to reproduce
1. Create a form with file input field and add the `required` attribute

## Expected result
File input to show the default field styling on page load. 
![ExpectedResult](https://user-images.githubusercontent.com/100657275/229072624-1c9778a2-79b5-4a69-a38c-3220d56ed990.png)

## Actual result
![ActualResult](https://user-images.githubusercontent.com/100657275/229072700-79b54f6d-2067-44ce-b4d8-cc074d812166.png)

## Testcase
[https://codepen.io/katedo/pen/ZEMNwEK](https://codepen.io/katedo/pen/ZEMNwEK)
```html
<div class="ui container">
  <form class="ui form">
    <div class="field required">
      <label>Field 1</label>
      <input type="file" name="field1" required>
    </div>
    
    <div class="field required">
      <label>Field 2</label>
      <div class="ui file input">
        <input type="file" name="field2" required>
      </div>
    </div>
    
    <button class="ui primary button">Submit</button>
  </form>
</div>
```

## Version

2.9.2

**Comment by lubber-de** [comment]
This is by design. By declaring the field as required it gets automatically invalid by the browser unless a file has been selected.
Thus it's a given hint, (like [formvalidation](https://fomantic-ui.com/behaviors/form.html) does) that you need to fill the field.
Basically, by giving a placeholder to input fields this is not happening. However, input fields do not support custom placeholders.

So, if you dont want this anyway, you could rebuild FUI and disable the [`@variationInputInvalid`](#2534) by setting it to false inside the [`variation.variables`](https://fomantic-ui.com/introduction/build-tools.html#variationvariables) file.

**Comment by lubber-de** [comment]
of course you can simply oiverride with some custom CSS in case you cannot or won't rebuild FUI
```CSS
.ui.form .field input[type="file"]:required:invalid,
.ui.file.input input[type="file"]:required:invalid {
  background: #fff;
    border: 1px solid rgba(34,36,38,.15);
    color: rgba(0,0,0,.87);
}
```

**Comment by jamessampford** [comment]
@lubber-de unless I'm missing something, it doesn't seem very user-friendly if a user has just loaded the page and the fields are highlighted as an error when the user hasn't done anything

Imagine a long form just opened up with many required fields, it'll make the user wonder what they've done wrong (which is nothing)

Will expect it to error if the user presses the submit button, but not beforehand

Is it not possible to trigger the lookup only after submitting the form?

**Comment by lubber-de** [comment]
I changed my mind and fixed that by #2751 which now fully supports native browser form validation.
Visual updates to native invalid fields will now only take place when the form was submitted at least once
